exp/ams.md

```diff
@@ -1,29 +1,30 @@
-The Anbox Management Service (AMS) sits at the heart of Anbox Cloud and handles all aspects of the application and container life cycle. It is responsible for managing containers, applications, addons, updates and more, ensuring high density, performance and fast container startup times.
+The Anbox Management Service (AMS) sits at the heart of Anbox Cloud and handles all aspects of the application and container life cycle. It is responsible for managing containers, applications, addons, updates and more, while simultaneously ensuring high density, performance and fast container startup times.
 
-AMS is usually managed through the command line interface (the Anbox Management Client - AMC), which can run either on the same machine as AMS or on a remote machine.
+## Interacting with AMS
+AMS is usually managed through the command line interface, the Anbox Management Client (AMC), which can run either on the same machine as AMS or on a remote machine.
 
-Since AMS exposes an HTTP interface, any tool can use the [AMS HTTP API](https://discourse.ubuntu.com/t/ams-rest-api-reference/17801) to interact with AMS. Both the AMC (when running remotely) and the [Anbox Application Registry (AAR)](https://discourse.ubuntu.com/t/application-registry/17761) use this mechanism. You can also develop your own client by using the [AMS SDK](https://discourse.ubuntu.com/t/ams-sdk-api-reference/17845).
+Since AMS exposes an HTTP interface, any tool can use the [AMS HTTP API](https://discourse.ubuntu.com/t/ams-rest-api-reference/17801) to interact with AMS. Both the AMC (when running as a remote client) and the Anbox Application Registry (AAR) use the AMS HTTP API to interact with AMS.
 
-<a name="security-certificates"></a>
-## Security certificates for remote clients
+You can also develop your own client by using the [AMS SDK](https://discourse.ubuntu.com/t/ams-sdk-api-reference/17845).
 
-Talking with AMS through HTTP requires a secure and trusted setup for communications, using TLS and [certificates](https://en.wikipedia.org/wiki/X.509).
+<a name="security-certificates"></a>
+### Security certificates for remote clients
 
-[note type="information" status="Note"]The AMC running on the same machine as AMS communicates with AMS through a socket, not through HTTP. Therefore, you do not need to worry about security certificates for local clients.[/note]
+If the AMC is running on the same machine as AMS, it communicates with AMS through a socket, not through HTTP. Therefore, you do not need to worry about security certificates for local clients. However, in case of remote clients,  interacting with AMS through HTTP requires a secure and trusted setup for communications, using TLS and [certificates](https://en.wikipedia.org/wiki/X.509).
```

Comment:
```suggestion
If the AMC is running on the same machine as AMS, it communicates with AMS through a UNIX domain socket, not through HTTP. Therefore, you do not need to worry about security certificates for local clients. However, in case of remote clients,  interacting with AMS through HTTP requires a secure and trusted setup for communications, using TLS and [certificates](https://en.wikipedia.org/wiki/X.509).
```